### PR TITLE
Build improvements

### PR DESCRIPTION
**Pull request by mimol91**
Add depedencies to docker
Update client npm
Update client configuration

Allow to customize build instance dir
Fix while unpacking zip (https://github.com/xsolve-pl/xsolve-feat/issues/23)

### CODE REVIEW

File: server/config/main.ini.dist [hunk 1]
```diff
@@ -9,6 +9,9 @@ port = 443
 [mongo]
 dsn = mongodb://user:password@host:27017/feat
 
+[build]
+instancesDirectory = /home/${user}/buildInstances
```
Comment (malef) [inline server/config/main.ini.dist]: Could we use some default that will point to a subdirectory inside the project root directory?

Comment (mimol91) [inline server/config/main.ini.dist]: good idea, I ll fix it

Comment (malef) [inline server/config/main.ini.dist]: Hello @mimol91 - any news on this maybe?

**Comment by mimol91** [comment]
Should be ok now.
Probably I will contribute less for next 2 months.

**Pull request closed without merging**